Jellyfin.Plugin.Anime/Jellyfin.Plugin.Anime.csproj

```diff
@@ -2,20 +2,22 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Jellyfin.Plugin.Anime</RootNamespace>
     <AssemblyVersion>1.0.0</AssemblyVersion>
     <FileVersion>1.0.0</FileVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="Configuration\configPage.html" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.2.0.1" />
+    <PackageReference Include="Jellyfin.Model" Version="10.2.0.1" />
```

Comment:
This isn't explicitly needed, as `Jellyfin.Controller` already includes it.

Reply:
But I don't see it hurting anything leaving it in there either, the plugin template has it, so if you feel like that shouldn't be standard then we should probably change that

Reply:
We are all about reducing unnecessary cruft though ;)

Reply:
I'll see about dropping the line if I get the chance